src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAX_NUMBER = 100;
+
+    public Ball getRandomBall() {
```

Comment:
The method `getRandomBall` should not be static. Static methods are generally considered bad practice in object-oriented programming because they hinder inheritance and polymorphism. Instead, you can create an instance of the `Lottery` class and call the method on that instance.